src/input_cp2k_pwdft.F

```diff
@@ -169,106 +169,112 @@ SUBROUTINE fill_in_section(section, section_name)
+                                      repeats=.FALSE., &
+                                      default_l_val=.TRUE., &
+                                      lone_keyword_l_val=.TRUE.)
+               ELSE
+                  CALL keyword_create(keyword, __LOCATION__, &
+                                      name=name1, &
+                                      description=TRIM(ADJUSTL(description)), &
+                                      usage=TRIM(ADJUSTL(usage)), &
+                                      repeats=.FALSE., &
+                                      default_l_val=.FALSE., &
+                                      lone_keyword_l_val=.TRUE.)
+               ENDIF
+            CASE (13)
+               CALL sirius_option_get_logical(section_name, name, lvec(1), vec_length)
+               DO j = 1, vec_length
+                  lvecl(j) = lvec(j)
+               ENDDO
                CALL keyword_create(keyword, __LOCATION__, &
                                    name=name1, &
                                    description=TRIM(ADJUSTL(description)), &
                                    usage=TRIM(ADJUSTL(usage)), &
-                                   repeats=.FALSE.)
-            END IF
-         CASE default
-         END SELECT
-         CALL section_add_keyword(section, keyword)
-         CALL keyword_release(keyword)
+                                   repeats=.FALSE., &
+                                   n_var=vec_length, &
+                                   default_l_vals=lvecl(1:vec_length))
+            CASE (4)
+               !     string need a special treatment because the parameters can only have dedicated values
+               default_string_val = CHAR(0)
+               CALL sirius_option_get_string(section_name, name, default_string_val)
+               default_string_val = TRIM(ADJUSTL(default_string_val))
+               CALL sirius_option_get_number_of_possible_values(section_name, name, num_possible_values)
+               IF (num_possible_values > 0) THEN
+                  DO j = 0, num_possible_values - 1
+                     possible_values(j + 1) = CHAR(0)
+                     CALL sirius_option_string_get_value(section_name, name, j, possible_values(j + 1))
+                     enum_i_val(j + 1) = j
+                  END DO
+                  CALL keyword_create(keyword, __LOCATION__, &
+                                      name=name1, &
+                                      description=TRIM(ADJUSTL(description)), &
+                                      usage=TRIM(ADJUSTL(usage)), &
+                                      repeats=.FALSE., &
+                                      enum_i_vals=enum_i_val(1:num_possible_values), &
+                                      enum_c_vals=possible_values(1:num_possible_values), &
+                                      default_i_val=0)
+               ELSE
+                  CALL keyword_create(keyword, __LOCATION__, &
```

Comment:
I think, this will still trigger the warnings from #317. You need to specify the type of the keyword, either explicitly via `type_of_var` or implicitly by providing a default value.

Reply:
Actually, what trigger it is somewhere else. I just escluded these two keywords from the list because they are useless for cp2k. Info about xc_functional are taken from libxc section, and memory_usage is useless for non devs.

I fixed it yesterday but I had a problem with github which means it did not go through. Should be fixed now.

Reply:
Yeah, Github is a bit flaky lately...

My point is that, every call to `keyword_create` should have either `default_?_val` or  `type_of_var` as an argument. 
And currently [this one](https://github.com/cp2k/cp2k/blob/bd5662a07fca9e0db6e8fa5041f9b7f1b32a08ed/src/input_cp2k_pwdft.F#L267-L271)  has neither.

Btw, once this is fixed, I'll turn [that warning](https://github.com/cp2k/cp2k/blob/master/src/input/input_keyword_types.F#L286) into an abort so our regtests can catch it.

Reply:
We should document that ;). I just fixed it.